Mathlib/LinearAlgebra/AffineSpace/AffineMap.lean

```diff
@@ -565,6 +565,9 @@ theorem lineMap_vsub_lineMap (p₁ p₂ p₃ p₄ : P1) (c : k) :
     lineMap p₁ p₂ c -ᵥ lineMap p₃ p₄ c = lineMap (p₁ -ᵥ p₃) (p₂ -ᵥ p₄) c :=
   ((fst : P1 × P1 →ᵃ[k] P1) -ᵥ (snd : P1 × P1 →ᵃ[k] P1)).apply_lineMap (_, _) (_, _) c
 
+@[simp] lemma lineMap_lineMap_right (p₀ p₁ : P1) (c d : k) :
```

Comment:
Can you add the left version too? Something like
```
lineMap (lineMap p₀ p₁ c) p₁ d = lineMap p₀ p₁ _
```

Reply:
I already tried adding the left version before PRing, but failed the algebra 🙈